src/components/Button/index.jsx

```diff
@@ -1,13 +1,24 @@
 import "./styles.scss";
 import PropTypes from "prop-types";
 
-const Button = (props) => {
-  return <button type={props.type} className='button'>{props.text}</button>;
+const Button = ({ type = "button", text, className, icon }) => {
+  return (
+    <button type={type} className={`button ${className || ""}`}>
+      {className && (
+        <span className="button__img">
+          <>{icon}</>
```

Comment:
This fragment (<>...</>) is unnecessary because `icon` is already a valid JSX element